README.md

```diff
@@ -1 +1,80 @@
-# Budget-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+#### Original design idea by **[Gregoire Vella on Behance](https://www.behance.net/gregoirevella)**, in association with the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) of the design.
+<br />
```

Comment:
- [ ] as the previous reviewer mentioned please add this appreciation to the Acknowledgments section at the bottom of the Readme file.

Reply:
Okay. Thank you.